web/src/app/[locale]/(app)/donations/event/page.tsx

```diff
@@ -0,0 +1,187 @@
+'use client'
+
+import { Box, Button, Flex, Grid, Heading, Image, Text } from '@chakra-ui/react'
+import { useRouter, useSearchParams } from 'next/navigation'
```

Comment:
```suggestion
import { useSearchParams  } from 'next/navigation'
import { useRouter } from '@/i18n/navigation'
```

use internationalized routing, in this way there are less redirects

Reply:
this comment applies to all files